extension/surveys/survey-generator.js

```diff
@@ -571,3 +571,45 @@ function getFormValues(questionArr, form) {
   }
   return responses;
 }
+
+/**
+ * Checks whether a form has any responses or text in it.
+ * @param {object} form The form to check.
+ * @returns {boolean} True if the form has content.
+ */
+function formHasContent(form) {
+  for (var i = 0; i < form.elements.length; i++) {
+    var element = form.elements[i];
+    switch (element.type) {
+      case 'checkbox':
+      case 'radio':
+        if (element.checked)
+          return true;
+        break;
```

Comment:
nit: This isn't in the style guide but a quick search at cs/ suggests that you don't need |break|s after returns.